### DISCUSSION TITLE
MyPamphlet-Blog/2023/09/16/mkdocs-material/

### DISCUSSION THREAD

**Discussion by giscus** [RonaldLN/MyPamphlet-Blog#1] [Announcements]
# MyPamphlet-Blog/2023/09/16/mkdocs-material/

https://ronaldln.github.io/MyPamphlet-Blog/2023/09/16/mkdocs-material/

**Comment by L-1124** [comment]
请问这段调的是哪里的字体
```css
  --md-typeset-kbd-color: hsla(var(--md-hue), 15%, 94%, 0.12);
  --md-typeset-kbd-accent-color: hsla(var(--md-hue), 15%, 94%, 0.2);
  --md-typeset-kbd-border-color: hsla(var(--md-hue), 15%, 14%, 1);
```

**Reply by RonaldLN** [reply]
我也不是很清楚😂我的暗色的主题css属性是直接复制mkdocs material官方默认暗黑主题 `slate` ，然后再在此基础上修改的，所以我只改了我当时看到的部分，其他不知道设置的是什么的css属性我就没有更改了😂
